### PR TITLE
Popup window Pull request

### PR DESCRIPTION
**Pull request by HERMON-1995**
- This project is all about building popup page using JavaScript dynamically in JS file. Here is some of my changes:

            > Created elements with JavaScript methods.
            > Created DOM events when clicked.
            > Selected elements using index of arrays.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,12 +11,11 @@
     <title>Hermons' Portfolio</title>
 </head>
 <body>
-  <div>
     <header class="header">
      <h1 class="logo">Welcome</h1>
      <nav class="site-nav">
```
Comment (topeogunleye) [inline index.html:16]: - [ ] As required, please kindly ensure that the mobile menu popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins). 
- Currently the mobile menu for your mobile is also showing on desktop. please fix it to have the same design as your desktop;

| Title  | Image Comparison |
| ------------- | ------------- |
| Your Design  |  image |
| Figma Design | image  |

File: index.html [hunk 2]
```diff
@@ -11,12 +11,11 @@
...
        <button class="hamburger"><div class="bar" title="bars"></div></button>
-    <ul class="nav-menu">
+     <ul class="nav-menu">
```
Comment (topeogunleye) [inline index.html:18]: - [ ] I kindly suggest that you ensure that your hamburger is responding to click and bringing up a modal. 
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- The popup is responsive, it fits mobile and desktop sizes properly.

File: index.html [hunk 3]
```diff
@@ -76,7 +75,7 @@ <h4 class="project-title-1">Multi-Post Stories<br>Gain+Glory</h4>
                  <li class="javascript">JavaScript</li>
```
Comment (topeogunleye) [inline index.html:75]: - [ ] Great job adding your project modals dynamically 👍, however you also need to add your project cards themselves dynamically as you did for the project modal.

- [ ] Kindly implement an array of objects to store your JavaScript objects containing the information for each project.

## EXAMPLE
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

- [ ] I kindly suggest that you ensure the Modal popup data is populated with the data from the projects array.

File: index.html (hunk 1 above)
Comment (topeogunleye) [inline index.html:16]: - [ ] I kindly suggest that you fix your mobile menu display on desktop as you currently have two menus displayed on desktop with one causing a giant horizontal scroll that makes the user not have a good experience browsing the website. please kindly fix it. I explain more in the video below:

https://www.loom.com/share/e8e9aca30c8243a8b73a610d9a6a61bf

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @HERMON-1995, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @HERMON-1995 

You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights
✔️ Descriptive PR 
✔️ your linters are all ok

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**